download_bancor_conversion_paths

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+DATAFILE=currencydata.json
+test -f $DATAFILE && echo "$DATAFILE exists" && exit
```

Comment:
Would be nice if this also checked for the existence of the `jq` command, and errored gracefully if it's not available

Reply:
added in latest commit